src/app/movies/[movieId]/page.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+async function getMovie(id: any) {
+  const res = await fetch(`https://plankton-app-xhkom.ondigitalocean.app/api/movies/${id}`);
+  return res.json();
+}
+
+export default async function movieInfo({ params: { movieId } }) {
+  console.log(`hej ${movieId}`);
+  const movieData = getMovie(movieId);
+  const [movie] = await Promise.all([movieData]);
```

Comment:
You have some unnecessary code in here.
This piece of code should be more than enough, promiseAll is used for running multiple promises and it's not needed here. 
```ts
const movie = await getMovie(movieId);
```

Reply:
Yes you are right. Have changed it now